spec/presenters/document/search_result_presenter_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+RSpec.describe Document::SearchResultPresenter do
+  let(:controller) { ActionView::TestCase::TestController.new }
+  let(:bl_response) { Europeana::Blacklight::Response.new(api_response, {}) }
+  let(:document) { bl_response.documents.first }
+  let(:basic_api_response) { JSON.parse(api_responses(:search)) }
+  let(:api_response) { basic_api_response }
+
+  describe '#hit_selector' do
+    subject { described_class.new(document, controller, api_response).hit_selector }
+
+    context 'with hit in API response' do
+      let(:api_response) do
+        basic_api_response.tap do |response|
```

Comment:
since we'll be adding the 'hits' profile by default to all searches I'd consider including it on the basic api response as opposed to modifying it on the fly here.

Reply:
But on the other hand, only the newspapers API will return hits...